src/Ledger/GovernanceActions.lagda

```diff
@@ -62,11 +64,10 @@ data GovAction : Type where
   Info             :                                             GovAction
 
 actionWellFormed : GovAction → Type
-actionWellFormed (ChangePParams x)  = ppdWellFormed x
-actionWellFormed (TreasuryWdrl x)   = 
-  (∀[ a ∈ dom x ] RwdAddr.net a ≡ NetworkId)
-  × (∃[ v ∈ range x ] ¬ (v ≡ 0))
-actionWellFormed _                  = ⊤
+actionWellFormed (ChangePParams x)  =  ppdWellFormed x
+actionWellFormed (TreasuryWdrl x)   =  ∀[ a ∈ dom x ] RwdAddr.net a ≡ NetworkId
+                                       × ∃[ v ∈ range x ] ¬ (v ≡ 0)
```

Comment:
This PR is not aimed at making the code look nicer, but this small change here makes the code more readable, and I won't remember to (and don't want to) make a separate PR for it later.